subprojects/test-runner/test-inhouse-runner/src/main/kotlin/com/avito/android/test/AbstractLaunchRule.kt

```diff
@@ -38,18 +38,20 @@ abstract class AbstractLaunchRule : TestRule {
 
     fun startFromHomeScreen() {
         beforeAppStart()
-        testContext.startActivity(Device.getLauncherIntentForAppUnderTest(testContext, appUnderTestContext))
```

Comment:
Can't start an activity from instrumentation context in Android 11:

```
java.lang.SecurityException: Permission Denial: package=<app>.test does not belong to uid=10154
	at android.os.Parcel.createExceptionOrNull(Parcel.java:2373)
	at android.os.Parcel.createException(Parcel.java:2357)
	at android.os.Parcel.readException(Parcel.java:2340)
	at android.os.Parcel.readException(Parcel.java:2282)
	at android.app.IActivityTaskManager$Stub$Proxy.startActivity(IActivityTaskManager.java:3696)
	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1723)
	at androidx.test.runner.MonitoringInstrumentation.execStartActivity(MonitoringInstrumentation.java:564)
	at android.app.ContextImpl.startActivity(ContextImpl.java:1023)
	at android.app.ContextImpl.startActivity(ContextImpl.java:994)
	at com.avito.android.test.AbstractLaunchRule.startFromHomeScreen(AbstractLaunchRule.kt:41)
	at ... <test>
	at java.lang.reflect.Method.invoke(Native Method)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.avito.android.test.AbstractLaunchRule$LaunchStatement.evaluate(AbstractLaunchRule.kt:35)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at androidx.test.internal.runner.TestExecutor.execute(TestExecutor.java:56)
	at androidx.test.runner.AndroidJUnitRunner.onStart(AndroidJUnitRunner.java:444)
	at com.avito.android.runner.InstrumentationTestRunner.onStart(InstrumentationTestRunner.kt:51)
	at com.avito.android.runner.InHouseInstrumentationTestRunner.onStart(InHouseInstrumentationTestRunner.kt:243)
	at android.app.Instrumentation$InstrumentationThread.run(Instrumentation.java:2205)
Caused by: android.os.RemoteException: Remote stack trace:
	at com.android.server.wm.ActivityTaskManagerService.assertPackageMatchesCallingUid(ActivityTaskManagerService.java:2598)
	at com.android.server.wm.ActivityTaskManagerService.startActivityAsUser(ActivityTaskManagerService.java:1081)
	at com.android.server.wm.ActivityTaskManagerService.startActivityAsUser(ActivityTaskManagerService.java:1072)
	at com.android.server.wm.ActivityTaskManagerService.startActivity(ActivityTaskManagerService.java:1047)
	at android.app.IActivityTaskManager$Stub.onTransact(IActivityTaskManager.java:1422)
```

Checked that at older API works also.